db/schema.rb

```diff
@@ -29,6 +29,19 @@
     t.index ["tenant_id"], name: "index_authentications_on_tenant_id"
   end
 
+  create_table "availabilities", force: :cascade do |t|
+    t.string "resource_type"
+    t.bigint "resource_id"
+    t.string "action", null: false
+    t.string "identifier", null: false
```

Comment:
we should add unique index on the first 4 columns

Reply:
:+1: done